contrib/kind.sh

```diff
@@ -603,7 +605,41 @@ create_kind_cluster() {
   if kind get clusters | grep ovn; then
     delete
   fi
+  if [[ "${KIND_LOCAL_REGISTRY}" == true ]]; then
+      # create registry container unless it already exists
+      if [ "$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)" != 'true' ]; then
```

Comment:
Right, let me check how it works with podman

Reply:
Looks like kind podman support is not GA, https://github.com/kubernetes-sigs/kind/blob/412a2adaeb3d86eadba93010cf72d97c1a0857c6/pkg/cluster/provider.go#L110

Do you know if much of the ovn-k community is using podman to start the kind with it ?

Reply:
Can we do a follow up with docker ? the local registry does not work out of the box replacing docker with podman also there is still parts of kind.sh that only works with docker.

Apart from that podman support at kind is not GA https://github.com/kubernetes-sigs/kind/blob/412a2adaeb3d86eadba93010cf72d97c1a0857c6/pkg/cluster/provider.go#L110

Reply:
I'm not a maintainer, but if it were up to me, no. This change is useless for people using podman, which is one of the [allowed/advertised](https://github.com/ovn-org/ovn-kubernetes/blob/master/docs/kind.md#run-the-kind-deployment-with-podman) container runtimes of the ovn-k kind integration.

Sure, podman support at kind is not GA, but I don't see why that matters since kind is not meant to run production workloads. 

It is possible to load the images using podman. The [multus repo does it](https://github.com/k8snetworkplumbingwg/multus-cni/commit/392726842fc19411834b7bd67fbbe1daf32c1aed).

Finally, afaict, slirp4netns is not needed, since afaiu, ovn-k in kind requires rootfull podman; from [its docs](https://github.com/ovn-org/ovn-kubernetes/blob/master/docs/kind.md#run-the-kind-deployment-with-podman):
> To deploy KIND however, you need to start it as root and then copy root's kube config to use it as non-root:

Reply:
Adapter the PR to work with "roorful" podman, thanks!